portal/site_persistence.py

```diff
@@ -382,6 +387,24 @@ def update_intervention(intervention_json):
                     "site_persistence: {}".format(repr(apptext)))
                 db.session.delete(apptext)
 
+        # ScheduledJobs shouldn't be order dependent, now is good.
+        jobs_seen = []
+        for j in objs_by_type['ScheduledJob']:
+            job = ScheduledJob.from_json(j)
+            db.session.add(job)
+            jobs_seen.append(j['name'])
+
+        # Delete any ScheduledJobs not named
+        if not keep_unmentioned:
+            jobs_seen.append("__test_celery__")
```

Comment:
@pbugni yea, check out https://github.com/uwcirg/true_nth_usa_portal/blob/develop/portal/celery_worker.py#L29 .

I wanted a test task that wouldn't be visible to users, but would help us debug on the backend to confirm that celery beat's working as intended for each instance (the job basically just updates its own `last_runtime` every hour).

This job will get recreated by the celery worker when it starts, if it doesn't already exist. But I added this line in here to handle the (very unlikely) scenario that we seed an instance *while* celery's running, so that celery isn't attempting to update a nonexistent job.